### PR TITLE
Add iOS chat demo

### PR DESCRIPTION
**Pull request by ziyiz-amzn**
**Description of changes:**
Add an iOS demo to demonstrate how to quickly add chat feature into iOS app using Amazon Chime SDK for messaging.

**Testing**

1. How did you test these changes? manually
2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it? n/a

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by zhinang-amazon — APPROVED** [review]

**Review by hokyungh — APPROVED** [review]

**Review by linsang21 — DISMISSED** [review]

**Review by devalevenkatesh — APPROVED** [review]

File: apps/chat-ios/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Amazon Chime SDK iOS Chat Demo
+
+## Overview
+
+By completing the steps below, you will learn how to run the Amazon Chime SDK iOS chat demo applications on your mobile device or on a simulator. This application enables users to send and receive messages/image attachments in chat channels in real time. This application is designed in a way that can be used as a starting point for your own application, or as a quick way to explore the features of the Amazon Chime SDK messaging. Please see the [Build chat applications in iOS and Android with Amazon Chime SDK messaging](https://aws.amazon.com/blogs/business-productivity/build-chat-applications-in-ios-and-android-with-amazon-chime-sdk-messaging) for more information.
```
Comment (devalevenkatesh) [inline apps/chat-ios/README.md:5]: The blog link is broken, could you please update it.

**Review by devalevenkatesh — CHANGES_REQUESTED** [review]
Need to fix the messaging SDK blog link in README.

File: apps/chat-ios/README.md (hunk 1 above)
Comment (devalevenkatesh) [reply apps/chat-ios/README.md:5]: The messaging SDK chat demo README calls Amazon Chime SDK for messaging and not Amazon Chime SDK messaging hence suggest to follow the same convention in this README as well as Android chat demo README.

Comment (ziyiz-amzn) [reply apps/chat-ios/README.md:5]: > The messaging SDK chat demo README calls Amazon Chime SDK for messaging and not Amazon Chime SDK messaging hence suggest to follow the same convention in this README as well as Android chat demo README.

The official name is Amazon Chime SDK messaging: https://docs.aws.amazon.com/chime/latest/dg/using-the-messaging-sdk.html
This blog post is reviewed by the author of web chat demo blog, Ryan, as well. The README is only a mini version of the blog. So `Amazon Chime SDK messaging` should be fine.

Comment (ziyiz-amzn) [reply apps/chat-ios/README.md:5]: > The blog link is broken, could you please update it.

The link will be live once publish, the url has been assigned in WordPress: https://aws.amazon.com/blogs/business-productivity/build-chat-applications-in-ios-and-android-with-amazon-chime-sdk-messaging/

**Comment by devalevenkatesh** [comment]
Can we merge both (android and ios) PRs close to the blog publishing time so that builders wont get confused on the blog link here?

But rest looks good.

**Review by hokyungh — APPROVED** [review]

**Review by devalevenkatesh — APPROVED** [review]

**Pull request merged**